numpy/distutils/misc_util.py

```diff
@@ -2326,6 +2326,43 @@ def get_info(name):
                 return g.get(name, g.get(name + "_info", {}))
 
             def show():
+                """
+                Print information about BLAS and LAPACK libraries in use.
```

Comment:
Or maybe

```suggestion
                Print information about libraries NumPy was built against.
```
Am I right in thinking this indicates what numpy actually uses? Or does it indicate what rebuilding numpy would use?
```

Reply:
That change looks good to me.  I don't know the answer to those questions;  I was assuming it shows what is being used by the numpy that ran `show_config()`.  I'll take a look at the `system_info` module now.

Reply:
Worth checking before talking my suggestion :)

Reply:
It looks like it searches the system for the available libraries.   Seems like a better name would be something like `show_system_info()` (which matches the underlying class that contains the code that does the work) or `show_available_library_info()`.  Anyway, I'll fix the text.